recipients_app/lib/view/widgets/welcome/phone_input_page.dart

```diff
@@ -18,13 +19,15 @@ class _PhoneInputPageState extends State<PhoneInputPage> {
   late final RoundedLoadingButtonController btnController;
```

Comment:
In the widgets (I also just learned) its better to assign them directly:

```
final btnController = RoundedLoadingButtonController();
final phoneNumberController = TextEditingController();
...
```

Reply:
@novas1r1 Do you have a link where your learning is described? :)